File: iOS/Objects/src/B4iProject/LS_frmsms.m

#import "iCore.h"
@interface LS_frmsms:NSObject
@end

@implementation LS_frmsms

- (void)LS_general:(B4ILayoutData*)views :(int)width :(int)height{
[B4ILayoutBuilder setScaleRate:0.3];
[B4ILayoutBuilder scaleAll:views :width :height];
[B4ILayoutBuilder setScaleRate:0.0];
[views get:@"btnproduts"].Width = (int)((50 / 100.0 * width));
[views get:@"btnsend"].Width = (int)((50 / 100.0 * width));
[views get:@"btnsend"].Left = (int)((50 / 100.0 * width));

}
@end

File: Android/Objects/src/com/apps/demo/designerscripts/LS_frmsms.java
package com.apps.demo.designerscripts;
import anywheresoftware.b4a.objects.TextViewWrapper;
import anywheresoftware.b4a.objects.ImageViewWrapper;
import anywheresoftware.b4a.BA;


public class LS_frmsms{

public static void LS_general(java.util.LinkedHashMap<String, anywheresoftware.b4a.keywords.LayoutBuilder.ViewWrapperAndAnchor> views, int width, int height, float scale) {
anywheresoftware.b4a.keywords.LayoutBuilder.setScaleRate(0.3);
anywheresoftware.b4a.keywords.LayoutBuilder.scaleAll(views);
anywheresoftware.b4a.keywords.LayoutBuilder.setScaleRate(0d);
views.get("btnproduts").vw.setWidth((int)((50d / 100 * width)));
views.get("btnsend").vw.setWidth((int)((50d / 100 * width)));
views.get("btnsend").vw.setLeft((int)((50d / 100 * width)));

}
}